app/resources/js/components/search-results-page.js

```diff
@@ -3,7 +3,6 @@ const ATTRIBUTES = [
   'id',
   'display_title',
   'date',
-  'result',
```

Comment:
Probably a stupid question - why is this gone?

Reply:
Or rather - was this unused before?

Reply:
It’s not gone completely, but it’s now only included when fetching data for `/votes`. When feting data for `/members/:id`, we retrieve the `members.${id}` attribute instead.

https://github.com/HowTheyVote/epvotes/pull/707/files#diff-e6903bf1e45a09bd1417c6051799566378f9a71210359b0673f6578263e8a194R111-R112

Reply:
The code is somewhat confusing though and could probably be simplified to something like this:

```js
const attributesToRetrieve = [...ATTRIBUTES, `members.${this.memberId}` || 'result'];
```

or

```js
const attributesToRetrieve = this.memberId
  ? [...ATTRIBUTES, `members.${this.memberId}`]
  : [...ATTRIBUTES, 'result']
```

Reply:
Got it, thanks. 
Personally, I did not find the other snippets to be easier to read than the code as is.